eks/karpenter-scaler.tf

```diff
@@ -0,0 +1,52 @@
+resource "aws_cloudformation_stack" "karpenter" {
+  count      = var.karpenter_enabled ? 1 : 0
+  name = "karpenter-stack"
+
+  parameters = {
+    ClusterName = aws_eks_cluster.cluster.name
+  }
+
+  template_body = file("${path.module}/karpenter.yml")
```

Comment:
Use https://registry.terraform.io/providers/hashicorp/http/latest/docs/data-sources/http to get the data
```
data "http" "karpenter_cloudformation" {
  url = "https://karpenter.sh/docs/getting-started/cloudformation.yaml"
}

template_body = http.karpenter_cloudformation.body
```

Reply:
I thought to make static, because it may change for each release